src/main/resources/bootstrap.yaml

```
@@ -1,11 +1,11 @@
 spring:
   cloud:
     propertiesvolume:
-      paths: /mnt/secrets/s2s,/mnt/secrets/ccd
+      paths: /mnt/secrets/ccd
```

you could add prefixed false here and then get rid of all the ccd prefixes in the aliases?
```suggestion
      prefixed: false
      paths: /mnt/secrets/ccd
```